README.md

```diff
@@ -11,6 +11,10 @@ Swiper component for React Native featuring **previews**, **snapping effect**, *
 
 -----
 
+### Do you want a better plugin? [Vote for React Native's feature requests](https://github.com/archriss/react-native-snap-carousel/issues/203)
```

Comment:
@florentroques Do you mind modifying it to the following? A small emoji and a bit of clarification cannot harm ;-)
```
### :sparkles: Do you want an even better plugin? [Vote for React Native's feature requests](https://github.com/archriss/react-native-snap-carousel/issues/203) to let the Facebook team know what they need to improve!
```

Reply:
sure